utils/ResponseWrapper.ts

```diff
@@ -0,0 +1,25 @@
+import http from 'http'
+import { ResponseProps } from '@types'
+import { ErrorText } from './Constants'
+export default function ResponseWrapper<T = unknown>(
+	res: ApiResponse,
+	{ code = 200, message, version = 2, data, errors }: ResponseProps<T>
```

Comment:
```suggestion
	{ code, message, version, data, errors }: ResponseProps<T> = { code: 200, version: 2 }
```

Reply:
This will cause code, version undefined when argument is passed